gnd/src/main/java/com/google/android/gnd/ui/editobservation/EditObservationFragment.java

```diff
@@ -284,7 +290,7 @@ private void onShowPhotoSelectorDialog(Field field) {
     RecyclerView recyclerView = addPhotoBottomSheetBinding.recyclerView;
     recyclerView.setHasFixedSize(true);
     recyclerView.setLayoutManager(new LinearLayoutManager(getContext()));
-    recyclerView.setAdapter(new AddPhotoDialogAdapter(listener));
+    recyclerView.setAdapter(new AddPhotoDialogAdapter(consumer));
```

Comment:
..then refer to it here as `new AddPhotoDialogAdapter(this::onSelectPhotoClick)`